index.html

```diff
@@ -0,0 +1,196 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+    </head>
+<body>
+    <div class="hdiv">
+        <header class ="head">Uthman</header>
+        <i class="fa-solid fa-bars menu-bar"></i> 
+    </div> 
+    <section class="section1">
+    <h1 class="h">I'm Uthman <br> Glad to see <br>you!</h1> 
+    <p class="info1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="lc info2">LET’S CONNECT</p>
+    <footer class="foot">
+        <div class="left">
+            <ul class="footer-items">
+            <li class="dotless twitter"><img src='images/Vector.svg' alt="twitter icon"></li>
+            <li class="dotless insta"><img class="iinsta" src='images/i.png'><img class="ninsta" src='images/n.png' alt="instagram icon"></li>
+            <li class="dotless m"><img src='images/M.svg' alt="m icon"></li>
+            <li class="dotless gh"><img src='images/gitHub.svg' alt="github icon"></li>
+            <li class="dotless peace"><img src='images/peace.svg' alt="peace icon"></li>
+            </ul>
+        </div>
+    </footer>
+    </section>
+    <section class="section2">
+        <div>
+            <section class="border-box">
+                <img src="images/Snapshoot Portfolio.png" class="snapshot">
+                <p class="head2">Tonic</p>
+                <div class="up1">
+                    <div class="p1">CANOPY</div>
+                    <div class="counter"></div>
+                    <div class="p2">Back End Dev</div>
+                    <div class="counter"></div>
+                    <div class="p3">2015</div>
+                </div>
+            </section>
+            <img src="" alt="">
+            <section>
+            <p class="p41">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="middle">
+            <div class="back-link1"><div class="link1">html</div></div>
+            <div class="back-link2"><div class="link2">css</div></div>
+            <div class="back-link3"><div class="link3">javaScript</div></div>
+            </div>
+            <button class="see-project1"><div class="see-project">See Project</div></button>
+            </section>
+        </div>
+        <section>
+            <div></div>
+            <div></div>
+        </section>
+    </section>
+    <section class="section3">
+        <div>
+            <section class="border-box">
+                <img src="images/Snapshoot Portfolio (1).png" class="snapshot">
+                <p class="head2">Multi-Post Stories</p>
```

Comment:
- [ ] Please make sure that the project title (e.g., "Tonic") uses a header tag `h2` or `h3` for semantic reasons :pray: